Legacy/bonej/pom.xml

```diff
@@ -116,13 +116,21 @@
             <groupId>sc.fiji</groupId>
             <artifactId>3D_Viewer</artifactId>
         </dependency>
+        <dependency>
+            <groupId>sc.fiji</groupId>
+            <artifactId>VIB_</artifactId>
+        </dependency>
```

Comment:
Check if VIB_ is still needed with `mvn dependency:analyze`

Reply:
You are right:
```
[WARNING] Unused declared dependencies found:
[WARNING]    sc.fiji:VIB_:jar:3.0.2:compile
```
also:
```
[WARNING] Unused declared dependencies found:
[WARNING]    org.scijava:vecmath:jar:1.6.0-scijava-2:compile
```
and:
```
[WARNING] Unused declared dependencies found:
[WARNING]    com.miglayout:miglayout-swing:jar:5.2:compile
```

Reply:
that last one about `miglayout` may require separate consideration.

Reply:
Miglayout IS needed runtime or GUI borks